### PR TITLE
Renew the deploy flow

### PR DESCRIPTION
**Pull request by unkcpz**
Tested at https://unkcpz.github.io/aiida-code-registry/

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -18,15 +18,22 @@ jobs:
     - uses: pre-commit/action@v2.0.0
 
   deploy-gh-pages:
+    if: github.repository == 'aiidateam/aiida-code-registry' # xxx: change to aiidateam after
```
Comment (yakutovicha) [inline .github/workflows/deploy.yml:21]:
```suggestion
    if: github.repository == 'aiidateam/aiida-code-registry'
```

File: .make_ghpages/index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<html>
+<body>
+
+<h1>Code registry</h1>
+
+<p><a href="/aiida-code-registry/database.json">Code registry for AiiDA 1.x</a></p>
+<p><a href="/aiida-code-registry/database_v2.json">Code registry for AiiDA 2.x</a></p>
+
+</body>
+</html>
```
Comment (yakutovicha) [inline .make_ghpages/index.html:11]:
```suggestion
</html>

```

File: .make_ghpages/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h1>Code registry</h1>
```
Comment (yakutovicha) [inline .make_ghpages/index.html:5]:
```suggestion
<h1>AiiDA codes and computers registry</h1>
```

**Review by yakutovicha — CHANGES_REQUESTED** [review]
Thanks, @unkcpz. Just a couple of minor changes.

**Comment by unkcpz** [comment]
@yakutovicha thanks! changed.

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**